### PR TITLE
[docs] Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `typos --format brief`

### CODE REVIEW

File: tests/unit_tests/repobee/test_main.py [hunk 1]
```diff
@@ -363,7 +363,7 @@ def test_does_not_log_error_when_command_is_used_incorrectly(mocker):
 
     with pytest.raises(SystemExit):
         # note that the category is misspelled
-        main.main("repbee isues -h".split())
+        main.main("repbee issues -h".split())
```
Comment (slarse) [inline tests/unit_tests/repobee/test_main.py:366]: This is intentionally misspelled as part of a regression test :)

```suggestion
        main.main("repbee isues -h".split())
```

**Review by slarse — CHANGES_REQUESTED** [review]
Hi! Thanks for you contribution, and sorry for the wait.

I just have one comment, one of the edits nullifies a regression test :)

**Review by slarse — APPROVED** [review]

**Comment by slarse** [comment]
Thanks! Docs building is failing but that seems unrelated, so I'll merge.

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**